server/midddleware/auth.go

```diff
@@ -124,3 +145,11 @@ func AuthFunction(ctx context.Context, jwtValidator *validator.Validator, config
 	// this should never happen.
 	return ctx, api.Errorf(api.Code_UNAUTHENTICATED, "You are not authorized to perform this action")
 }
+
+func getFullMethodName(ctx context.Context) (string, bool) {
```

Comment:
not for this diff, but move this method in request_metadata.

Reply:
yes you are right this is more suitable there. moved it to request_metadata.